### ISSUE TITLE
Command fails if subcommand contains double quote (")

### ISSUE THREAD

**Issue by jlacour31** [dita-ot/dita-ot#4525]
## Expected Behavior
The dita.bat command should succeed.

## Actual Behavior
The command fails.

## Possible Solution
It can be resolved by using enabledelayedexpansion:
```
setlocal enabledelayedexpansion
set command=%DITA_CMD_SUBCOMMAND%
if "!command!" == "install" goto checkJava
if "!command!" == "uninstall" goto checkJava
```
## Steps to Reproduce
1. Open the command in DITA-OT /bin folder
2. Run `dita "--input=test\that\fails.ditamap"`

## Copy of the error message, log file or stack trace
=test\that\fails.ditamap"" was unexpected at this time.

## Environment
* DITA-OT version: 4.2.3
* Operating system and version: Windows
* How did you run DITA-OT? `dita` command

**Comment by jelovirt** [comment]
None of the core developers has a Windows machine nor do we have access to one. If you're able to test the suggested fix, please open a pull-request with the fix.

**Comment by jlacour31** [comment]
@jelovirt from which branch do I need to make the PR? I tested the fix on my Windows machine and it did work.

**Comment by raducoravu** [comment]
@chrispy-snps maybe you can double check the approved fix on your Windows machine.

**Comment by chrispy-snps** [comment]
@raducoravu - the fix looks good to me.

Before fix (`8de366a88a45cb0b287cf06701eccba2adc197f1`):

```
PROMPT> dita-ot-ORIG\build\tmp\dist\bin\dita.bat "--input=NOTFOUND.ditamap" -f html5
=NOTFOUND.ditamap"" was unexpected at this time.
```

After fix (`2e127334845021981cf45be6bcbb265131f393a1`):

```
PROMPT> dita-ot-NEW\build\tmp\dist\bin\dita.bat "--input=NOTFOUND.ditamap" -f html5
Error: Build failed with an exception: [DOTA069F] The input resource 'file:/C:/Users/chrispy/Downloads/test/dita-ot-NEW/build/tmp/dist/plugins/org.dita.base/NOTFOUND.ditamap' cannot be located or read. Make sure 'file:/C:/Users/chrispy/Downloads/test/dita-ot-NEW/build/tmp/dist/plugins/org.dita.base/NOTFOUND.ditamap' exists and that you have permission to access it.
```
